pkgs/by-name/py/python/python-modules/build/default.nix

```
@@ -28,7 +28,7 @@ buildPythonPackage rec {
     owner = "pypa";
     repo = "build";
     rev = "refs/tags/${version}";
-    hash = "sha256-SGWpm+AGIfqKMpDfmz2aMYmcs+XVREbHIXSuU4R7U/k=";
+    hash = "sha256-qlt5+3XFesF9UuvJ+D4F/1KVZ3cov6sTitjufwAq73Q=";
```

I think upgrading to `1.2.1` would be a better idea.
```suggestion
    hash = "sha256-G0g+1v19sQMUuQlZKGELZOwwX07i7TIdEdaYzr8bKtI=";
```

---

That would also fix the initial problem so that's fine with me